ch_tools/common/logging.py

```diff
@@ -0,0 +1,213 @@
+"""
+Logging module.
+"""
+
+import inspect
+import logging
+from logging import (  # noqa # pylint:disable=unused-import
+    CRITICAL,
+    DEBUG,
+    ERROR,
+    FATAL,
```

Comment:
Unused

Reply:
It's for addressing https://github.com/yandex/ch-tools/pull/141#discussion_r1570180044